janus_core/cli/geomopt.py

```diff
@@ -33,6 +33,61 @@
 app = Typer()
 
 
+def _set_minimize_kwargs(
+    minimize_kwargs: dict[str, Any],
+    traj: Optional[str],
+    vectors_only: bool,
+    pressure: float,
+) -> None:
+    """
+    Set minimize_kwargs dictionary values.
+
+    Parameters
+    ----------
+    minimize_kwargs : dict[str, Any]
+        Other keyword arguments to pass to geometry optimizer.
+    traj : Optional[str]
+        Path if saving optimization frames.
+    vectors_only : bool
```

Comment:
`vectors_only` isn't the clearest name to me as to what that means. The help explains it, but I think `cell_only` would possibly be better since the atomic positions are also vectors.

Reply:
yes this is a tough one since cell_only will mean for sure angles too.. cell_vectors_only maybe?

Reply:
`cell_lengths`? `lattice_parameters`?

Reply:
cell_lengths has my vote.

Reply:
Opened as a new issue